lms/djangoapps/courseware/module_render.py

```diff
@@ -450,6 +451,17 @@ def handle_grade_event(block, event_type, event):  # pylint: disable=unused-argu
             descriptor.location,
         )
 
+        # Send a signal out to any listeners who are waiting for score change
+        # events.
+        SCORE_CHANGED.send(
+            sender=None,
+            points_possible=event['max_value'],
+            points_earned=event['value'],
+            user=user,
+            course=course_id,
+            usage=descriptor.location
+        )
```

Comment:
I considered that, but my thinking was that while we only care about actual changes for the LTI work there may be other uses for the signal later when somebody wants to get a signal for every submission. I figured it'd be better to err on the side of sending more information.